CHANGELOG.md

```diff
@@ -37,8 +37,11 @@ Markdown Spec](https://github.github.com/gfm/).
 ### Changed
 - Upgraded to Solr 8.3 - this includes our docker-compose setup as well as the
   way Solr is configured
+  - The schema.xml and solrconfig.xml files have been removed in leiu of using
```

Comment:
....hate to say this, but "lieu", right?

Reply:
Fixed